lnd_test.go

```diff
@@ -8724,8 +8726,56 @@ func testRouteFeeCutoff(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("unable to update chan policy: %v", err)
 	}
 
-	// Wait for Alice to receive the channel update.
-	time.Sleep(5 * time.Second)
+	// assertChanPolicy is a helper closure that ensures a policy has the
+	// correct parameters.
+	assertChanPolicy := func(policy *lnrpc.RoutingPolicy, baseFeeMSat,
+		feeRate int64, timeLockDelta uint32) {
+
+		if policy.FeeBaseMsat != baseFeeMSat {
+			t.Fatalf("expected base fee %d, got %d", baseFeeMSat,
+				policy.FeeBaseMsat)
+		}
+		if policy.FeeRateMilliMsat != feeRate {
+			t.Fatalf("expected fee rate %d, got %d", feeRate,
+				policy.FeeRateMilliMsat)
+		}
+		if policy.TimeLockDelta != timeLockDelta {
+			t.Fatalf("expected base fee %d, got %d", timeLockDelta,
+				policy.TimeLockDelta)
+		}
+	}
+
+	// waitForChanUpdate is a helper closure that will wait for an update
+	// for a specific channel and assert its routing policy.
+	waitForChanUpdate := func(graphUpdates chan *lnrpc.GraphTopologyUpdate,
```

Comment:
There's a `waitForChannelUpdate` elsewhere in this file. Is it possible to unify them, or are they checking different criteria?

Reply:
Fixed.